components/Starknet/modules/architecture-and-concepts/pages/network-architecture/messaging-mechanism.adoc

```diff
@@ -226,9 +227,22 @@ Where:
 
 [NOTE]
 ====
-In an `l1_handler` transaction, the first element of the calldata is always the Ethereum address of the sender.
+In a L1 handler transaction, the first element of the calldata is always the Ethereum address of the sender.
 ====
 
+[NOTE]
+====
+Since L1 handler transactions are not initiated by an account, invoking link:https://github.com/starkware-libs/cairo/blob/2203a47f8a098cd4718d03bd109ca014049419e7/corelib/src/starknet/info.cairo#L49[`get_caller_address()`] or similar account-related functions will return the address `0x0`.
```

Comment:
```suggestion
Since L1 handler transactions are not initiated by an account, invoking link:https://github.com/starkware-libs/cairo/blob/2203a47f8a098cd4718d03bd109ca014049419e7/corelib/src/starknet/info.cairo#L49[`get_caller_address()`] or similar account-related functions returns the address `0x0`.
```